qutebrowser/browser/hints.py

```diff
@@ -833,7 +833,7 @@ def filter_hints(self, filterstr):
         for elems in self._context.elems.values():
             try:
                 if (filterstr is None or
-                        str(elems.elem).lower().startswith(filterstr)):
+                        filterstr.lower() in str(elems.elem).lower()):
```

Comment:
Good idea! However I suggest using [str.casefold](https://docs.python.org/3.4/library/stdtypes.html#str.casefold) instead of str.lower. :)